CONTRIBUTING.md

```diff
@@ -0,0 +1,63 @@
+# Contributing
+
+Want to contribute? Great! There are many ways to give back to the project, whether it be writing new code, fixing bugs, or just reporting errors. All forms of contribution are encouraged!
+
+## Reporting issues
+
+Notice something amiss? Have an idea for a new feature? Feel free to to write an [Issue](https://docs.github.com/en/free-pro-team@latest/github/managing-your-work-on-github/about-issues) on the GitHub repository about anything that you feel could be fixed or improved. Examples include:
+
+- Unclear documentation
```

Comment:
Might be worth creating common issue templates as well that correspond with these.

Reply:
I'll make a follow-up! The `.github` repo should be a good place for it (that's where organization-wide docs live)